### DISCUSSION TITLE
NCICT3.0.20240626

### DISCUSSION THREAD

**Discussion by choonsiklee** [ncidose/ncidosetools#28] [Announcements]
We fixed a bug concerning x-ray spectrum index in NCICT3.0.20240229 and released NCICT3.0.20240626. Apologies about the inconvenience.

**Comment by mkb600** [comment]
Hi @choonsiklee, I have noticed a bug on this version where the effective dose is fixed based on phantom. The organ doses change appropriately, but not the effective dose. I have tested both the mac and pc versions.

**Reply by choonsiklee** [reply]
@mkb600 Can you please download the current version from Code section and try again? I think there was some glitch while the new version was uploaded to GitHub repository.

**Reply by mkb600** [reply]
Thank you, that has corrected the issue.
